media/css/mozorg/home/home-promo.less

```diff
@@ -1042,24 +1042,6 @@ html[lang|="en"] {
 
 
 /****************************************************************************/
-// ViewSource Conf
-.promo-large-landscape.viewsource {
-    background-color: #00b085;
-
-    h2 {
-        padding-top: 120px;
-        .at2x('/media/img/home/voices/promos/viewsource/logo.png', 280px, 55px);
```

Comment:
True, but that's probably a ways off and we'll almost certainly have a new home page by then. View Source might even have a new logo by then for all we know. Deleting the images needn't be a blocker for merge, but it's nice to keep things tidy.